docs/UserGuide.md

```diff
@@ -318,6 +318,10 @@ inputs that ArtBuddy requires to execute some commands properly. For instance, t
 needs to know the name of the customer which you'll need to pass into ArtBuddy as a command parameter.
 <br><br>For ArtBuddy to understand these command parameters, you need to attach a ['prefix'](#glossary) in front of
 each of these parameters.
+<br><br>If a parameter is optional, for example address in customer, it will be indicated as <code>[a/ADDRESS]</code> with
+square brackets around it to indicate optionality.
+<br><br>If a parameter can take multiple values, for example tags in customer, it will be indicated as <code>[t/TAGS]...</code> with
+three dots after it to indicate multiple values.
```

Comment:
These points are already in the "Using this Guide" section (Conventions used in command formats). So perhaps we should remove it here? Else it might be a little wordy.

Reply:
Hmmm, because this section defines the command parameter, I feel it okay to repeat it shortly once again.

Reply:
My only concern is that repetition is one of the bugs that can be reported for UG (see below), and because 'unnecessary' is very vague, I think it'd be safer to just avoid repetitions in general especially for large chunks of text.

Also, one of Ms Serene's feedback was that parts of our UG are kinda wordy so I think it'd be good to make sure we  make things concise.

I agree that the command formatting is relevant to the command parameter though, but perhaps a better fix might be to use a callout instead and reference it to the command formatting section so that readers can reference the section if they want to?

<img width="734" alt="Screenshot 2022-11-06 at 9 36 23 PM" src="https://user-images.githubusercontent.com/79991214/200174030-8080d611-c262-4c4c-ac9c-481b8f65080b.png">